apps/wallet/src/ui/app/redux/slices/app/index.ts

```diff
@@ -29,7 +30,8 @@ const initialState: AppState = {
     navVisible: true,
     activeOrigin: null,
     activeOriginFavIcon: null,
-    isAppViewPopup: null,
+    isAppViewPopup: true,
```

Comment:
we could remove the `isAppViewPopup` boolean all together and just use a check similar as we do for tabs:
```
 const isTabView = useAppSelector(
        (state) => state.app.extensionViewType === ExtensionViewType.Tab,
    );
```
ex. in `AccountsFinderIntroPage.tsx` and  `WalletSettingsMenuList.tsx`
instead of   `  const isAppPopup = useAppSelector((state) => state.app.isAppViewPopup);`
we could add:
```
 const isAppPopup = useAppSelector(
        (state) => state.app.extensionViewType === ExtensionViewType.Popup,
    );
```